OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -1051,9 +1048,8 @@ void Nudge(Actor self)
 				.Rotate(WRot.FromFacing(self.World.SharedRandom.Next(256)));
 			var target = Target.FromPos(self.CenterPosition + offset);
 
-			self.CancelActivity();
-			self.SetTargetLine(target, Color.Green, false);
-			self.QueueActivity(new Fly(self, target));
+			self.QueueActivity(true, new Fly(self, target));
```

Comment:
Same here, also shouldn't this be `false` then? Using that QueueActivity overload with `true` basically just calls `QueueActivity(Activity)`, making the `true` redundant.

Reply:
This should be `false` of course, which makes it a very minor cleanup.